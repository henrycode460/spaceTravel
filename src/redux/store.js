import { configureStore } from '@reduxjs/toolkit';
import { combineReducers } from 'redux';
import missionsReducer from './missions/missions';
import rockertsReducer from './rockets/rockets';

const rootStore = combineReducers({
   missions: missionsReducer,
   rockets: rockertsReducer,
});

const store = configureStore({
  reducer: rootStore,
});

export default store;
