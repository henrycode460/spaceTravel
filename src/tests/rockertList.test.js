/* eslint-disable no-undef */
import TestRenderer from 'react-test-renderer';
import RockertList from '../components/rockertList.js';
import { Provider } from 'react-redux';
import store from '../redux/store.js'

it('Testing rockert component:', () => {
  const tree = TestRenderer.create(
    <Provider store={store}>
      <RockertList />
    </Provider>).toJSON();
  expect(tree).toMatchSnapshot();
});