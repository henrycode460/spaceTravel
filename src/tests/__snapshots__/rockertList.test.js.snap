// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing rockert component: 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "gap": "30px",
      "marginBottom": "50px",
    }
  }
>
  <img
    alt="rocket"
    style={
      Object {
        "width": "250px",
      }
    }
  />
  <div>
    <h3 />
    <p>
       
    </p>
    <button
      className="join show"
      onClick={[Function]}
      style={
        Object {
          "background": "rgba(0, 102, 255, 0.89)",
          "border": "none",
          "color": "white",
          "padding": "10px",
          "width": "150px",
        }
      }
    >
      Reserve Rocket
    </button>
    <button
      className="leave hide"
      onClick={[Function]}
      style={
        Object {
          "border": "solid 1px gray",
          "color": "gray",
          "padding": "10px",
          "width": "180px",
        }
      }
    >
      Cancel Resservation
    </button>
  </div>
</div>
`;

exports[`Testing rocket component: 1`] = `
<div>
  <img 
    
  />
    <div
      
    >
      <h3>

      </h3>
        <p>
          </p>

          <button>
          Reserve Rocket
          </button>

           <button>
           Cancel Resservation
          </button>
 
</div>
`;
